packages/trpc/server/routers/viewer/organizations/bulkDeleteUsers.handler.ts

```diff
@@ -57,13 +57,55 @@ export async function bulkDeleteUsersHandler({ ctx, input }: BulkDeleteUsersHand
     },
   });
 
+  const removeManagedEventTypes = prisma.eventType.deleteMany({
+    where: {
+      userId: {
+        in: input.userIds,
+      },
+      parent: {
+        team: {
+          OR: [
+            {
+              parentId: currentUserOrgId,
+            },
+            { id: currentUserOrgId },
```

Comment:
Do we really need this? as  

```
owner       User?   @relation("owner", fields: [userId], references: [id], onDelete: Cascade)
  userId      Int?
```
  
 onDelete: Cascade is already there

Reply:
The `onDelete: Cascade` rule would trigger deletion when the user entry is being removed from the database. However, in this case, we are **not** deleting the user from the database; we are only removing the relationship between the user and the organization

https://github.com/calcom/cal.com/blob/6e22e06fa90f578693c0c0e20ccae518fec09338/packages/trpc/server/routers/viewer/organizations/bulkDeleteUsers.handler.ts#L45-L58

The user record still exists in the database. Cascade only works when the user row itself is deleted @Udit-takkar